Tests from lib_io_file/fileiotest.fz using unicode:

🧪 exists: false
🧪 was created
🧪 exists: true
🧪/📄 exists: false
🧪/📄 was created
🧪/📄 exists: true
stat resolve=true : 🧪/📄 size is 25
stat resolve=false: 🧪/📄 size is 25
0
unit
1
unit
file content bytes: [240, 159, 145, 139, 240, 159, 143, 162, 240, 159, 167, 145, 226, 128, 141, 240, 159, 146, 187, 226, 152, 175, 239, 184, 143]
file content is: 👋🏢🧑‍💻☯️
🧪/📄 was deleted
🧪/📄 exists: false
🧪 exists: true
✨✨✨ exists: false
🧪 is now: ✨✨✨
✨✨✨ exists: true
🧪 exists: false
✨✨✨ was deleted
✨✨✨ exists: false


Tests from lib_io_dir/lib_io_dir.fz using unicode:

unit
--error: an error occurred while creating the following directory: "⚗️🧫"--
--error: no more directory entries--
1️⃣📜
2️⃣🗒️
